### DISCUSSION TITLE
Total System Failure

### DISCUSSION THREAD

**Discussion by reformedcigars** [simple-login/app#1063] [Q&A]
I have no idea what is going on, but both of my domains that were once verified with my SimpleLogin account are in total failure with this service. As far as I can tell, nothing is wrong with my domains. I just set both of them up within the last 3 months and everything was working fine until 24 hours ago. I haven’t changed anything on my end. Things just stopped working, that is, no aliases will forward. Now, my domains won't even re-verify the DNS settings that were once verified.

I’m getting absolutely no response from anyone at SimpleLogin using the contact form. The level of customer service for paid accounts is a joke. I am at a loss and this becoming detrimental to my business. Any ideas?

**Comment by nguyenkims** [comment]
Sorry for the inconvenience. Can you send me the support ticket number so I can speed up its resolution?

SimpleLogin uses Cloudflare to run DNS check and it should be one of the most reliable DNS services.

**Reply by reformedcigars** [reply]
This issue has since been resolved. However, the modes of support and response times with SimpleLogin need to be improved. Hopefully, the move to Proton will do this. Thanks for reaching out.
